src/main/java/uk/gov/hmcts/reform/civil/enums/GAJudicialHearingType.java

```diff
@@ -7,9 +7,9 @@
 @RequiredArgsConstructor
 public enum GAJudicialHearingType {
 
-    IN_PERSON("In person"),
-    VIDEO("Video"),
-    TELEPHONE("Telephone");
+    IN_PERSON("in person"),
+    VIDEO("via video hearing (Teams)"),
+    TELEPHONE("via Telephone (BT Meet Me)");
```

Comment:
cant see these changes requested anywhere in the requirement ticket ? this would affect other pdfs as well not just hearing form